.mk/k8s.mk

```diff
@@ -160,7 +161,7 @@ k8s-proxy-nsmgr-save:  $(addsuffix -save,$(addprefix ${CONTAINER_BUILD_PREFIX}-,
 .PHONY: k8s-proxy-nsmgr-load-images
 k8s-proxy-nsmgr-load-images:  $(addsuffix -load-images,$(addprefix ${CLUSTER_RULES_PREFIX}-,$(PROXY_NSMGR_CONTAINERS)))
 
-VPPAGENT_DATAPLANE_CONTAINERS = vppagent-dataplane vppagent-dataplane-dev
+VPPAGENT_DATAPLANE_CONTAINERS = vppagent-dataplane
```

Comment:
I would bet that the lack of the dev slims down the buidl time.

Reply:
not much, but a bit yes.